src/prompt.rs

```diff
@@ -3,10 +3,10 @@ use tera::{Context, Error};
 use std::collections::HashMap;
 use tera::Tera;
 
-pub fn format_prompt(prompt: &str, map: HashMap<&str, &str>) -> Result<String, Error> {
+pub fn format_prompt(prompt: &str, lang: &str, map: HashMap<&str, &str>) -> Result<String, Error> {
```

Comment:
Don't add a function argument here, define a `language` property in the `map` variable and use the tera templating logic for this